templates/confidential-computing/index.html

```diff
@@ -0,0 +1,287 @@
+        </div>
+        <hr />
+        <div class="row">
+          <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
+            <p class="u-no-margin--bottom">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/8e5cc412-AWS.svg",
+                alt="AWS logo",
+                height="22",
+                width="40",
+                hi_def=True,
+                loading="lazy" ) | safe
+              }}
+            </p>
+          </div>
+          <div class="col-4 col-medium-3 col-small-2">
+            <p><a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snp-requirements.html">AMD SEV SNP with Ubuntu 23.10 on AWS</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-12">
+        <p class="p-heading--2">
+          <a href="https://ubuntu.com/engage/introduction-confidential-computing">Download our guide to confidential computing ›</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip u-no-padding--top">
+    <div class="row--50-50">
+      <div class="col">
+        <h2>A solid foundation of your zero trust strategy</h2>
+      </div>
+      <div class="col">
+        <p>
+          With confidential computing, you can remove the privileged systems
+          software from your trusted computing base, reduce your attack surface,
+          and get a remotely verifiable cryptographic guarantee before trusting
+          any platform with your data.
+        </p>
+        <p>
+          <a href="https://ubuntu.com/blog/build-foundation-zero-trust-strategy-ubuntu-confidential-computing">Learn
+            more about zero trust and confidential computing ›</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip u-no-padding--top">
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Confidential computing in your private data centre</h2>
+      </div>
+      <div class="col">
+        <p>
+          Combine good data governance practices with the latest advances in
+          privacy-enhancing computation. Use confidential computing to protect
+          the confidentiality and integrity of the sensitive data hosted on your
+          on-premises servers, using hardware-rooted primitives beyond
+          traditional measures.
+        </p>
+        <p>
+          <a href="https://canonical.com/blog/why-do-you-also-need-confidential-computing-for-your-private-datacenter">Why
+            use confidential VMs in your private datacenter? ›</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep u-no-padding--top">
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Learn more about confidential computing</h2>
```

Comment:
We should add a break after "about" to avoid text orphans 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/1e381f8c-ef66-4531-a952-a5701568b13a)

Reply:
I'm not sure if we need to do that tbh.  To my eye it looks fine together with other sibling sections. @lyubomir-popov wdyt?